app/models/gogovan_order.rb

```diff
@@ -4,22 +4,26 @@ class GogovanOrder < ActiveRecord::Base
 
   has_one :delivery, inverse_of: :gogovan_order
 
+  before_create :generate_uuid
   after_commit :start_polling_status, on: [:create]
   before_destroy :cancel_order, if: :pending?
 
-  def self.save_booking(booking_id)
-    create(status: 'pending', booking_id: booking_id)
-  end
-
   def self.place_order(user, attributes)
     attributes = set_vehicle_type(attributes) if attributes['offerId']
     Gogovan.new(user, attributes).get_order_price
   end
 
   def self.book_order(user, attributes)
+    order = create(status: 'pending')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.